mbedtls/src/lib.rs

```diff
@@ -7,6 +7,8 @@
  * according to those terms. */
 
 #![deny(warnings)]
+// to be removed after migration to rustc 1.70.., issue #277
+#![allow(ambiguous_glob_reexports)]
```

Comment:
The combination of `#![deny(warnings)]` and `#![allow(ambiguous_glob_reexports)]` gives an error with older versions of Rust:
```
error: unknown lint: `ambiguous_glob_reexports`
```
However, with `#![allow(unknown_lints)]` inserted between those two attributes it seems to work nicely with Rust 1.65, at least, and perhaps it would for everyone that way, so consider adding that line.